sentry.gradle

```diff
@@ -169,7 +215,8 @@ gradle.projectsEvaluated {
             group = 'sentry.io'
 
             delete sourcemapOutput
-            delete "$buildDir/intermediates/assets/release/index.android.bundle.map" // react native default bundle dir
+            delete "$buildDir/intermediates/assets/release/index.android.bundle.map"
```

Comment:
H: The `release` part should use a var because this depends on the `variant` most likely, similar to how its done for other files.

Reply:
There is a source map deleted right above it and then there is this one written explicitly, I don't really know why.

Reply:
Not sure I understand your comment here, mind clarifying?

Reply:
[On this line](https://github.com/getsentry/sentry-react-native/pull/2606/files/c727f94e4b4ad137fd7d1d69822414437ee37640#diff-01c94c11105e37de1b11a82973b605f625fff5959b0386c32fd1e42a14325208R217) the correct map will be deleted.

And then there is the next line with the hard codded path.

Reply:
nvm, you just changed the comment, so all good